homeassistant/components/honeywell/const.py

```diff
@@ -9,7 +9,8 @@
 DEFAULT_HEAT_AWAY_TEMPERATURE = 61
 CONF_DEV_ID = "thermostat"
 CONF_LOC_ID = "location"
-TEMPERATURE_STATUS_KEY = "outdoor_temperature"
-HUMIDITY_STATUS_KEY = "outdoor_humidity"
-
+OUTDOOR_TEMPERATURE_STATUS_KEY = "outdoor_temperature"
+OUTDOOR_HUMIDITY_STATUS_KEY = "outdoor_humidity"
+CURRENT_TEMPERATURE_STATUS_KEY = "current_temperature"
+CURRENT_HUMIDITY_STATUS_KEY = "current_humidity"
```

Comment:
We typically want these to be in the platform they are used if not multi-platform usage

Reply:
Moved the new platform-specific constants into the sensor file.  Reopened the PR for review.